examples/counter/containers/CounterApp.js

```diff
@@ -1,23 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
I'd do it differently. Let's encourage separation of “smart” and “dumb” components. See https://github.com/gaearon/react-redux#dumb-component-is-unaware-of-redux and https://github.com/gaearon/react-redux#smart-component-is-connect-ed-to-redux.